src/components/TextAreaAutoSize/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import classNames from 'classnames';
+import TextArea from 'react-textarea-autosize';
+import { ValidationMessage, Variant } from '@edx/paragon';
+
+import './TextAreaAutoSize.scss';
+
+const TextAreaAutoSize = ({
```

Comment:
is there a reason why this wouldn't be in paragon? it doesn't seem specific to code assignment